R/readUNFCCC_NDC.R

```diff
@@ -69,6 +71,24 @@ readUNFCCC_NDC <- function(subtype, subset) {
 
     x <- as.magpie(input, spatial = "ISO_Code", temporal = "Target_Year")
 
+     } else {
```

Comment:
Could you fix indentation of the code? Can be done automatically in RStudio using Ctrl+A (mark all code), then Ctrl+I (fix indentation).